Sources/Jobs/AnyDecodableJob.swift

```diff
@@ -0,0 +1,62 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2024 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+/// Add codable support for decoding any JobInstance
+internal struct AnyDecodableJob: DecodableWithUserInfoConfiguration, Sendable {
+    typealias DecodingConfiguration = JobRegistry
+
+    init(from decoder: Decoder, configuration register: DecodingConfiguration) throws {
+        let container = try decoder.container(keyedBy: _JobCodingKey.self)
+        let key = container.allKeys.first!
```

Comment:
Can you explain the superdecoder dance you're doing here? Is this supposed to decode siblings?

Reply:
Is it possible to get rid of the force-unwrap in trade for throwing an error?

Reply:
Added comment and got rid of force unwrap